scene/resources/default_theme/default_theme.cpp

```diff
@@ -883,7 +883,7 @@ void fill_default_theme(Ref<Theme> &theme, const Ref<Font> &default_font, const
 	theme->set_stylebox("bg", "GraphEdit", make_stylebox(tree_bg_png, 4, 4, 4, 5));
 	theme->set_color("grid_minor", "GraphEdit", Color(1, 1, 1, 0.05));
 	theme->set_color("grid_major", "GraphEdit", Color(1, 1, 1, 0.2));
-	theme->set_constant("bezier_len_pos", "GraphEdit", 80 * scale);
+	theme->set_constant("bezier_len_position", "GraphEdit", 80 * scale);
```

Comment:
There is `bezier_len_neg`, so I assume `pos` in this case means 'positive'. Not sure if you'd want to refactor theme constants.